Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -385,9 +401,38 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     ) {
         updateNavigationTitle()
     }
-
+    
+    open func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        if typingMembers.contains(where: { member in member.id == self.channelController.client.currentUserId }) {
+            let typingMembersWithoutCurrentUser = typingMembers.filter { $0.id != self.channelController.client.currentUserId }
+            showTypingIndicatorIfNeeded(typingMembers: typingMembersWithoutCurrentUser)
+        } else {
+            // Current user is not member of the room but can see that someone is actually typing
+            showTypingIndicatorIfNeeded(typingMembers: typingMembers)
+        }
```

Comment:
I don't understand the logic here 🤔 Why should it matter if the current user is a member or not? In no case I want to show him as a typing user. Isn't the following enough?
```swift
let typingMembersWithoutCurrentUser = typingMembers
    .filter { $0.id != self.channelController.client.currentUserId }
showTypingIndicatorIfNeeded(typingMembers: typingMembers)
```

Reply:
Yes, the following is enough, the solution above is a bit more complex than it needs to be...